README.md

```diff
@@ -24,6 +23,7 @@ pip install shippo
 ```
 <!-- End SDK Installation [installation] -->
 
+
```

Comment:
in the below section on example usage, I think we should update the example as follows
```
res = s.addresses.list_addresses(page=1, results=25)
```
because users should not generally be sending the version on each request